src/styles/product-card.css

```diff
@@ -0,0 +1,101 @@
+.card {
+  width: 200px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  padding-left: 16px;
+  padding-right: 16px;
+
+  --main-accent-color: #060b35;
+  --secondary-color: #616070;
+}
+
+.card__img {
+  margin-top: 32px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-top: 40px;
+  color: var(--main-accent-color);
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-color);
+  margin-top: 4px;
+  width: 100%;
+}
+
+.card__feedback {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-accent-color);
+  display: flex;
+  justify-self: center;
+  align-self: center;
+}
+
+.card__rating {
+  display: flex;
+  margin-top: 16px;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.card__price {
+  display: flex;
+  margin-top: 24px;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.card__cost {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary-color);
+}
+
+.card__value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main-accent-color);
+}
+
+.card__buy {
+  display: flex;
+  margin-top: 16px;
+}
+
+.card__button {
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  background-color: #00acdc;
+  border: none;
```

Comment:
Add default transparent border of the same width, and change its color on :hover

Reply:
I didnt understand what you mean(

Reply:
9 point from [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)